backend/experiment/templates/csv-export.html

```diff
@@ -38,7 +38,7 @@
         margin-right: 2em;
     }
 </style>
-<h1>Export results for experiment: {{experiment.name}}</h1>
+<h1>Export results for block: {{experiment.name}}</h1>
```

Comment:
It still says `experiment.name` though 🤔

Reply:
Yes, you are a bit too fast ;-)
Now called `exp_block.name` as `block` is a reserved name in django templates.